src/Tizen.WebView/Tizen.WebView/PolicyDecisionEventArgs.cs

```diff
@@ -0,0 +1,206 @@
+    {
+        private IntPtr _handle;
+
+        internal PolicyDecisionEventArgs(IntPtr handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Gets the Url.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Url
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_url_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Gets the scheme.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string Scheme
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_policy_decision_scheme_get(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Ignores the action which triggers this decision.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Ignore()
+        {
+            Interop.ChromiumEwk.ewk_policy_decision_ignore(_handle);
+        }
+
+        /// <summary>
+        /// Accepts the action which triggers this decision.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public void Use()
+        {
+            Interop.ChromiumEwk.ewk_policy_decision_use(_handle);
+        }
+
+        /// <summary>
+        /// Checks if frame requested in policy decision is main frame.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsMainFrame()
+        {
+            return Interop.ChromiumEwk.ewk_policy_decision_is_main_frame(_handle);
+        } 
+    }
+
+    /// <summary>
+    /// This class is derived from PolicyDecisionEventArgs.
+    /// This class provides the properties for New Window Policy of WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class NewWindowPolicyEventArgs : PolicyDecisionEventArgs 
+    {
+        private IntPtr _handle;
+        internal NewWindowPolicyEventArgs(IntPtr handle) : base(handle)
+        {
+            _handle = handle;
+        }
+        
+        internal static NewWindowPolicyEventArgs CreateFromSmartEvent(IntPtr data, IntPtr obj, IntPtr info)
+        {
+            return new NewWindowPolicyEventArgs(info);
+        }
+    }
+
+
+    /// <summary>
+    /// This class is derived from PolicyDecisionEventArgs.
+    /// This class provides the properties for Navigation Policy of WebView.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class NavigationPolicyEventArgs : PolicyDecisionEventArgs
+    {
+        private IntPtr _handle;
+        internal NavigationPolicyEventArgs(IntPtr handle) : base(handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Checks Navigation Type Is Link Clicked.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsLinkClicked
```

Comment:
Plz revert this change, NavigationType should be provided.

Reply:
Done